src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/WorkloadCalculationEventListener.kt

```diff
@@ -22,7 +23,7 @@ class WorkloadCalculationEventListener(
     val availableHours = workloadCalculationEvent.additionalInformation.availableHours
     val staffCode = workloadCalculationEvent.personReference.identifiers.find { it.type == "staffCode" }!!.value
     val teamCode = workloadCalculationEvent.personReference.identifiers.find { it.type == "teamCode" }!!.value
-    val staffGrade = communityApiClient.getStaffSummaryByCode(staffCode).map { it.grade }.block()!!
+    val staffGrade = communityApiClient.getStaffByCode(staffCode, StaffSummary::class.java).map { it.grade }.block()!!
```

Comment:
It looks to me like Staff (DeliusStaff) has a grade, and that is all we are using here. 

Can't we simply have CommunityApiClient return Staff (DeliusStaff) and do away with the generics? Looks like it suits both usages I can see here.

Reply:
Ah, I have now seen the rest of the PR. 😄